app/src/main/java/com/kickstarter/ui/adapters/DiscoveryPagerAdapter.java

```diff
@@ -48,14 +50,27 @@ public CharSequence getPageTitle(final int position) {
   }
 
   /**
+   * Passes along root categories to its fragment position to help fetch appropriate projects.
+   */
+  public void takeCategoriesForPosition(final @NonNull List<Category> categories, final int position) {
```

Comment:
A review of how `fragmentManager` works: not all 5 sorts fragments are guaranteed to exist at any time. We pass along the position here from the Activity to ensure the categories are passed to a fragment that actually exists.